notebooks/work-in-progress/play_with_cems_crosswalk.ipynb

```diff
@@ -109,16 +108,16 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "Retrieving https://zenodo.org/api/deposit/depositions/6633770 from zenodo\n",
-      "Retrieving https://zenodo.org/api/files/4f9ac0dc-a9b4-4d2b-9e0c-c0e97a2fc7f6/datapackage.json from zenodo\n",
-      "Retrieving https://zenodo.org/api/files/4f9ac0dc-a9b4-4d2b-9e0c-c0e97a2fc7f6/epacems_unitid_eia_plant_crosswalk.zip from zenodo\n",
+      "Retrieving https://zenodo.org/api/deposit/depositions/7063255 from zenodo\n",
+      "Retrieving https://zenodo.org/api/files/afca4bd7-d94f-4af3-b1f3-abaeaec6fa68/datapackage.json from zenodo\n",
+      "Retrieving https://zenodo.org/api/files/afca4bd7-d94f-4af3-b1f3-abaeaec6fa68/epacamd_eia.zip from zenodo\n",
```

Comment:
If we're done with this WIP notebook do we want to remove it?

Reply:
Sure